src/EventManagement.Web/Pages/Admin/Events/Details.cshtml

```diff
@@ -302,6 +373,29 @@
         }
     };
 
+    $('#btn-save-product-changes').on('click', function(){
+        const data = {
+            registrationId: registrationId,
+            products: $('.product-input-group').get()
+                        .map(function(d) { 
+                            const productId = $(d).find('.input-prodid').first().val();
+                            return {
+                                id: productId, 
+                                quantity: $(d).find('.input-quantity').first().val(),
+                                variant: $(d).find('input[name="variant-radio-'+ productId +'"]:checked').val()
+                            }
+                        })
+        };
+        console.log(JSON.stringify(data))
+        $.postJSON('/api/v0/orders/update-order', data)
+            .done(function(){
+                toastr.success('Update. Refresh the page to see changes.');
```

Comment:
Or perhaps a .  `$('#participants-table').bootstrapTable('refresh', null);`?

Reply:
But isn't that the same as a page reload? If the admin scrolled down halfway, he'll have to do that again after each change.